### PR TITLE
CLI loading for SC datasets

### PR DESCRIPTION
**Pull request by pcm32**
This PR allows the SC data production to become independent of the web application for loading ONLY, by using the Java cli module.

### CODE REVIEW

File: bin/load_experiment_web_cli.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+java_opts="$java_opts -Djdbc.max_pool_size=2"
+java_opts="$java_opts -Dserver.port=8888"
+# This turns off some extensions for large vector calculations
+# which are bit irrelevant to the current task and give issues on
+# running this through an emulated container on M1
+java_opts="$java_opts -XX:UseAVX=0"
+
+# Generate JSONL files from bulk experiments
+echo "PATH: "$PATH
+
```
Comment (pcm32) [inline bin/load_experiment_web_cli.sh:54]: This can go.

Comment (pcm32) [reply bin/load_experiment_web_cli.sh:54]:
```suggestion
```

File: bin/load_experiment_web_cli.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
```
Comment (pinin4fjords) [inline bin/load_experiment_web_cli.sh:11]: Could we have a comment as to what this script is doing? Is it ultimately just creating a record in the experiments table based on MAGE-TAB content the CLI knows how to find?

Comment (pcm32) [reply bin/load_experiment_web_cli.sh:11]: Plus creating the expDesign files, which is what produces trouble with memory on tomcat. I have added to the env file for scxa prod all the needed variables for this to work.

**Comment by pcm32** [comment]
This is accompanied by https://github.com/ebi-gene-expression-group/atlas-prod/pull/245

**Review by pinin4fjords — APPROVED** [review]
Bit of an ITYWFI, but looks good

**Pull request merged**